app/styles/home.scss

```diff
@@ -147,11 +152,16 @@ button.small {
       margin: 0;
       padding: 0 15px;
       width: 33.33%;
+      min-height: 788.283px;
```

Comment:
This is an oddly specific number. Where did it come from?

Reply:
Ah, forgot to add a comment. It is the height of the list when it contains 10 items (which is always the case from what I can tell).

Reply:
This number seems wrong. Since it's dependent on the user's zoom setting and installed fonts, it should probably be measured in ems. For example, on my Windows machine, using Chrome, it's 774.261. I default to 110% zoom though. If I bump it down to 100%, it's 767.938.

Reply:
That's weird - Firefox and Chrome on Linux always show the same height (in px) regardless of zoom. Will look into it.

Reply:
They definitely shouldn't be. Zoom by definition changes the height in pixels.